.gitignore

```diff
@@ -10,9 +10,16 @@
 
 # production
 /build
+server.js
+firebase.js
+Blog.tsx
+Blog.module.css
 
+ 
 # secrets
 **/service-account.json
+**/ServiceAccountKey.json
+ServiceAccountKey.json
```

Comment:
I believe these credentials are compromised. Google reported it was seen here: https://github.com/danesh-23/samwise-positivity/blob/837f85081b9bde1e0012d1a9790c947d785cf001/ServiceAccountKey.json
Please generate a service account and replace the existing one with that.

Reply:
Ahh I see, I shall generate a new one and actually I can completely remove it this time since we don't need it for now.